### PR TITLE
fixed cyclic lr state dict

### PR DESCRIPTION
**Pull request by shaydeci**

### CODE REVIEW

File: src/super_gradients/training/utils/checkpoint_utils.py [hunk 1]
```diff
@@ -1598,3 +1597,16 @@ def load_pretrained_weights_local(model: torch.nn.Module, architecture: str, pre
...
+
+def get_scheduler_state(scheduler) -> Dict:
+    """
+    Wrapper for getting a torch lr scheduler state dict, resolving some issues with CyclicLR
+    (see https://github.com/pytorch/pytorch/pull/91400)
+    :param scheduler: torch.optim.lr_scheduler._LRScheduler, the scheduler
+    :return: the scheduler's state_dict
+    """
+    state = scheduler.state_dict()
+    if isinstance(scheduler, CyclicLR) and int(torch.version.__version__.split(".")[0]) < 2:
```
Comment (BloodAxe) [inline src/super_gradients/training/utils/checkpoint_utils.py:1610]: Same https://github.com/Deci-AI/super-gradients/pull/1468/files#r1329656508

Comment (shaydeci) [reply src/super_gradients/training/utils/checkpoint_utils.py]: fixed

**Review by BloodAxe — APPROVED** [review]
LGTM

**Review by BloodAxe — APPROVED** [review]
LGTM
